frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -105,16 +107,19 @@ const ProjectAccess: React.FC<ProjectAccessProps> = ({ formName, namespace, role
   return (
     <>
       <PageHeading>
-        Project Access allows you to add or remove a user&apos;s access to the project. More
-        advanced management of role-based access control appear in{' '}
-        <Link to={`/k8s/ns/${getActiveNamespace()}/${RoleModel.plural}`}>Roles</Link> and{' '}
-        <Link to={`/k8s/ns/${getActiveNamespace()}/${RoleBindingModel.plural}`}>Role Bindings</Link>
-        . For more information, see the{' '}
-        <ExternalLink
-          href={`${openshiftHelpBase}authentication/using-rbac.html`}
-          text="role-based access control documentation"
-        />{' '}
-        .
+        <Trans t={t} ns="devconsole">
+          Project Access allows you to add or remove a user&apos;s access to the project. More
```

Comment:
shouldn't this be
```suggestion
          Project access allows you to add or remove a user&apos;s access to the project. More
```

Reply:
I took a closer look at the capitalization guidelines and glossaries and it looks like lowercase is appropriate here.

Reply:
Thanks, Have Updated.